landing-pages/site/content/en/blog/airflow-survey.md

```diff
@@ -0,0 +1,333 @@
+|Original Airflow Graphical User Interface, CLI       |117|37.99%|
+|API                                                  | 60|19.48%|
+|Original Airflow Graphical User Interface, CLI, API  | 32|10.39%|
+|Custom (own created) Airflow Graphical User Interface| 25|8.12% |
+
+It’s visible that usage of CLI goes in pair with using Airflow web UI. Our
+survey included some UX related questions to allow us to understand how users
+use Airflow webserver.
+
+**What do you use the Graphical User Interface for?**
+
+![](plot1.png)
+
+**What do you use CLI for?**
+
+![](plot2.png)
+
+**In Airflow, which UI view(s) are important for you?**
+
+![](plot3.png)
+
+Here we see that majority uses Web UI mostly for monitoring purposes:
+
+- Monitoring DAGs
+- Accessing logs
+
+An interesting result is that many people seem to do not use backfilling as
+there’s no other way than to do it by CLI.
+
+**What executor type do you use?**
+
+|          |No.|  %   |
+|----------|---|------|
+|Celery    |138|44.81%|
+|Local     | 85|27.60%|
+|Kubernetes| 52|16.88%|
+|Sequential| 22|7.14% |
+|Other     | 11|3.57  |
+
+The other option mostly consisted of information that someone uses few types or is
+migrating from one executor to another. What can be observed is an increase in usage
+of Local and Kubernetes executors when compared to results from earlier [survey done
+by Ash](https://ash.berlintaylor.com/writings/2019/02/airflow-user-survey-2019/).
+
+**Do you use Kubernetes-based deployments for Airflow?**
+
+|                                                                     |No.|  %   |
+|---------------------------------------------------------------------|---|------|
+|No - we do not plan to use Kubernetes near term                      | 88|28.57%|
+|Yes - setup on our own via Helm Chart or similar                     | 65|21.10%|
+|Not yet - but we use Kubernetes in our organization and we could move| 61|19.81%|
+|Yes - via managed service in the cloud (Composer / Astronomer etc.)  | 45|14.61%|
+|Not yet - but we plan to deploy Kubernetes in our organization soon  | 42|13.64%|
+|Other                                                                |  7|2.27% |
+
+The most interesting thing is that there’s nearly 30% of users who do not use Kubernetes
+and they are not going to move. This means we should keep Kubernetes in mind when working
+on Airflow 2.0. On the other hand, almost 70% of the users either already use Kubernetes
+or it’s a viable option for them.
+
+**Do you combine multiple DAGs?**
+
+|                                 |No.|  %   |
+|---------------------------------|---|------|
+|No, I don't combine multiple DAGs|127|41.23%|
+|Yes, through SubDAG              | 73|23.70%|
+|Yes, by triggering another DAG   | 72|23.38%|
+|Other                            | 36|11.69%|
+
+In the other category, there were 9 people who explicitly mentioned using
+`ExternalTaskSensor` and I think it could be treated as running subDAGs by
+triggering other DAGs.
```

Comment:
Wow. I'm one of the `ExternalTaskSensor` guys 😁 
very interesting to see that this is not the way to do this.